doc/user/content/releases/v0.27.0.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Materialize v0.27.0"
+date: 2022-11-01
+---
+
+v0.27.0 is the first cloud-native release of Materialize. It contains
+substantial breaking changes from [v0.26 LTS].
+
+## Changes
+
+* Add [clusters](/sql/create-cluster) and [cluster
+  replicas](/sql/create-cluster-replica/). Each cluster is an isolated, highly
+  available pool of compute resources.
```

Comment:
I'm not sure this is an accurate description of clusters?  A cluster is an isolated, logical container of zero or more compute resources.  Clusters can be made highly available through replication.

Maybe a nit -- don't feel too strongly about this -- but want our description to be consistent with documentation.

Reply:
Yeah, fair. It's hard to phrase so I'll just strike that line for now! These release notes are more like TODOs that we should fill in over the next three months rather than the polished notes we [used to write up](https://materialize.com/docs/release-notes/#v0.23.0).